### PR TITLE
Specify botorch version

### PR DESCRIPTION
**Pull request by Alnusjaponica**
## Motivation
Fix [the CI error](https://github.com/optuna/optuna-examples/actions/runs/3640122560) by the recent update of `botorch`.

## Description of the changes
Specify the version of `botorch` < 8.0.0 as [#PR4228](https://github.com/optuna/optuna/pull/4228).

### CODE REVIEW

File: multi_objective/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-botorch
+botorch>=0.4.0,< 0.8.0
```
Comment (toshihikoyanase) [inline multi_objective/requirements.txt:1]: A nit
```suggestion
botorch>=0.4.0,<0.8.0
```

File: multi_objective/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 fvcore
 torch
 torchaudio
-torchvision
+torchvision
```
Comment (toshihikoyanase) [inline multi_objective/requirements.txt:5]: I don't think this line has any changes. Do you have any ideas?

**Review by toshihikoyanase — COMMENTED** [review]
Thank you for your PR. The change looks good to me, except for a minor suggestion.

File: multi_objective/requirements.txt (hunk 2 above)
Comment (Alnusjaponica) [reply multi_objective/requirements.txt:5]: You' right. This part should not be changed.

**Review by toshihikoyanase — APPROVED** [review]
Thank you for your update. LGTM!

**Pull request merged**